website/content/api-docs/jobs.mdx

```diff
@@ -122,6 +122,10 @@ The table below shows this endpoint's support for
 
 - `Job` `(Job: <required>)` - Specifies the JSON definition of the job.
 
+- `Submission` `(JobSubmission: <optional>)` - Specifies the original HCL/HCL2/JSON
+  definition of the job. This data is useful for reference only, it is not considered
+  for the actual registration of `Job`.
```

Comment:
```suggestion
  for the actual scheduling of `Job`.
```

A bit pedantic so nbd either way. I think arguably it _is_ considered by the Job.Register operation, just not for any operational logic. Just saying "scheduling" might make the distinction more clear even if it's ignored by more than just scheduling logic.